Update [04092017] Tolits

tolits123 opened the pull request:
Update Student (Requirements)
Masked (StudentCreate_frm)
StudentCreate null pic
DeleteStudent sql Query
Class_frm (For testing)

ashlrem commented:
LAN Connection - OK
Splash Screen Test DB Connection - OK
Add Admin first admin account - OK
Add Admin account - OK
Add Cashier account - OK
Add Registrar account - OK
Update Admin account -  OK
Update Registrar Account - OK
Update Cashier Account - OK
Delete Admin Account - OK
Delete  Registrar Account - OK
Delete Cashier Account - OK
View Admin Account - OK
View Registrar Account - OK
View Cashier Account - OK
Add Class - OK
Add Student Info - OK
AdminCreate_1, Choose Picture, gets empty fields when clicking register - OK
Masked Password Fields not clickable - OK
Employee id text fields not clickable - OK
Picture retains after creating admin account - OK
Update status combobox fixed to Active upon load - OK
Add Blocked option to combobox - OK
Search Admin - OK
Search Registrar - OK
Search Cashier - OK
Update Student, NSO not showing uploaded even if it is not null on database. *Check baptismal as well - OK
Update Student, GREEN BACKGROUND if file uploaded to DB, RED if Not yet uploaded file, Clickable *NSO and Baptismal - OK
Update Student, Confirmation message on message box after updating student info, grammar (Student Info updated successfully.) - OK
Update Class SQL QUERY - OK
Delete Student Info message box if student not found, SQL Query, student table instead of subject_table - OK
Create database table for student info archive - OK

Search Student -
Update Student Info - OK - But needs improvement, check UPDATE.
Develop new form for search student info archive


UPDATE:

Update Registrar shows timestamp of created time of data to database - 
Admin menu strip Add Class instead of Add Subjects - 

Delete Class SQL QUERY -
Delete Class Close Button goes to registrar screen -

Add Student Birthplace Masked textfield not clickable - change to textfield
Add Student Religion Masked textfield not clickable - change to textfield
Add Student School year Masked textfield not clickable - change to textfield
Add Student Age Masked textfield not clickable - change to textfield
Add Student Address Masked textfield not clickable - change to textfield
Add Student ALL FAMILY INFO Textfield  Masked textfield not clickable - change to textfield

Update Student, dialog box opened once
Update Student, NSO, Baptismal, Upload Card, Upload Form137, Upload Good Moral Label, and button at bottom

Delete Class, search button change sql query, delete subject button change sql query, close button goes to registar, message box change message
Requirements Groupbox info remains after clicking enroll button, must clear also
Enroll student, no picture selected does fatal error
Enroll student, remove text fields under picture box, use variables instead


Cashier Screen, picture not loading after login
Cashier Add Payment, id lace and patch, must be enabled as well
Cashier Add Payment, needs improvement
Cashier Update Payment, not tested
Cashier Delete Payment, not tested

Registrar Student Create, Masked text boxes not clickable, change to textbox
Registrar Student Create, add photo cannot be null
Registrar Student Update, error
Registrar Student Delete, error

tolits123 committed:
Update [04092017] Tolits

Update Student (Requirements)
Masked (StudentCreate_frm)
StudentCreate null pic
DeleteStudent sql Query
Class_frm (For testing)
